app/components/inventory/rank-select.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+import SelectValues from 'hospitalrun/utils/select-values';
+
+const c = Ember.computed;
```

Comment:
that being said, if this project is using the latest shims

https://github.com/ember-cli/ember-cli-shims/blob/master/app-shims.js#L194-L214

one can:

```js
import computed from 'ember-computed'
```